modules/logging-release-notes-6-0-0.adoc

```diff
@@ -0,0 +1,97 @@
+[id="logging-release-notes-6-0-0_{context}"]
+= Logging 6.0.0
+
+This release includes link:https://access.redhat.com/errata/RHBA-2024:6693[OpenShift Logging Bug Fix Release 6.0.0]
+
+[id="logging-release-notes-6-0-0-compatibility-support-matrix"]
+= Compatibility and support matrix
+
+In the table, components are marked with the following statuses:
+
+[horizontal]
+TP:: Technology Preview
+GA:: General Availability
+
+The components or features in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] status are experimental and are not intended for production use.
+
+[NOTE]
+====
+To know about compatible {product-title} versions for {logging-uc} releases, see link:https://access.redhat.com/product-life-cycles?product=Red%20Hat%20OpenShift%20Logging[Product Life Cycles].
+====
+
+.Components support matrix
+[options="header"]
+|===
+
+| {logging-uc} Version 6+| Component Version
+
+| Operator | `eventrouter` | `logfilemetricexporter` | `loki` | `lokistack-gateway` | `opa-openshift` | `vector`
+
+|6.0 | 0.4 (GA) | 1.1 (GA) | 3.1.0 (GA) | 0.1 (GA) | 0.1 (GA) | 0.37.1 (GA)
+
+|===
+
+[id="logging-release-notes-6-0-0-removal-notice"]
+== Removal notice
+
+* With this release, logging no longer supports the `ClusterLogging.logging.openshift.io` and `ClusterLogForwarder.logging.openshift.io` custom resources. Refer to the product documentation for details on the replacement features. (link:https://issues.redhat.com/browse/LOG-5803[LOG-5803])
+
+* With this release,logging no longer manages or deploys log storage (such as Elasticsearch), visualization (such as Kibana), or Fluentd-based log collectors. (link:https://issues.redhat.com/browse/LOG-5368[LOG-5368])
+
+[NOTE]
+====
+In order to continue to use Elasticsearch and Kibana managed by the elasticsearch-operator, the administrator must modify those object's ownerRefs before deleting the ClusterLogging resource.
+====
+
+[id="logging-release-notes-6-0-0-enhancements"]
+== New features and enhancements
+=== Log Collection
+
+* This feature introduces a new architecture for Red Hat OpenShift Logging by shifting component responsibilities to their relevant operators, such as for storage, visualization, and collection. It introduces the `ClusterLogForwarder.observability.openshift.io` API for log collection and forwarding. Support for the `ClusterLogging.logging.openshift.io` and `ClusterLogForwarder.logging.openshift.io` APIs, along with the Red Hat managed Elastic stack (Elasticsearch and Kibana), is removed. Users are encouraged to migrate to the Red Hat `LokiStack` for log storage. Existing managed Elasticsearch deployments can be used for a limited time. Automated migration for log collection is not provided, so administrators need to create a new ClusterLogForwarder.observability.openshift.io specification to replace their previous custom resources. Refer to the official product documentation for more details. (link:https://issues.redhat.com/browse/LOG-3493[LOG-3493])
```

Comment:
I feel we should add some links in this bullet:
- "Red Hat `LokiStack` for log storage": Link to the OCP docs for setting up loki-operator
- "Elasticsearch deployments can be used for a limited time": Link to a deprecation notice of ES, how long is limited time?
- "Refer to the official product documentation for more details.": Link to the migration procedure

Reply:
I suppose bullet points will be useful, if there's a flow to what is being written. I tried writing a removal notice in bullet points earlier and it didn't work out as it somewhat changed the meaning or made it more confusing. Hence, ignoring this for now.